### DISCUSSION TITLE
Andina TV [BO]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#395] [Show and tell]
### Channel ID

AndinaTV.bo

### Stream URL

https://tv.mediacp.eu:8081/andinatv/index.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Stream does not work: geo-blocked?

**Comment by freearhey** [comment]
https://tv.mediacp.eu:8081/andinatv/index.m3u8
<img width="1405" alt="image" src="https://user-images.githubusercontent.com/7253922/232254374-f443dc88-2f4f-4474-a97d-1dc8f287979c.png">

**Comment by freearhey** [comment]
https://tv.mediacp.eu:8081/andinatv/index.m3u8 (offline)
<img width="346" alt="image" src="https://user-images.githubusercontent.com/7253922/233799800-3ac2cbb4-728e-4d83-afb1-f60d9ddb7d13.png">
